README.md

```diff
@@ -56,7 +56,7 @@ make start
 The server and client will be started:
 
 - API documentation available on <http://localhost:3000/api>
-- Client available on <http://localhost:3001/>
+- Client available on <http://localhost:3002/>
```

Comment:
This change was made because I couldn't make the proxy work. Just to keep README up to date with what works for me